scripts/run-unit-tests.sh

```diff
@@ -11,7 +11,7 @@ start-ganache
 # run unit tests
 echo "running unit tests..."
 rm -f coverage.txt
-go test ./... -v -short -timeout=30m -count=1 -covermode=atomic -coverprofile=coverage.txt
```

Comment:
As of go 1.10 you can use the `-coverpkg` flag I added to get cross-package coverage stats. Project wide, it didn't make a huge difference (1.4%), but it now shows coverage in `protocol/xmrmaker/claim.go` that we get from the daemon package.